### PR TITLE
Remove old Calendar dependency from RH-Management

### PR DESCRIPTION
**Pull request by Rdidmohamedelhadi**

### CODE REVIEW

File: component/pom.xml [hunk 1]
```diff
@@ -40,9 +45,23 @@
...
-
-    </dependencies>
+		<dependency>
+			<groupId>javax.jcr</groupId>
+			<artifactId>jcr</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.exoplatform.jcr</groupId>
+			<artifactId>exo.jcr.component.core</artifactId>
+			<version>6.2.x-SNAPSHOT</version>
```
Comment (ahamdi) [inline component/pom.xml:55]: Could you use variables for version tags?

Comment (Rdidmohamedelhadi) [reply component/pom.xml:55]: ok

File: pom.xml [hunk 2]
```diff
@@ -25,9 +25,10 @@
   </scm>
   <properties>
     <juzu.jackson.version>1.2.0</juzu.jackson.version>
-    <org.exoplatform.commons.version>6.0.x-SNAPSHOT</org.exoplatform.commons.version>
-    <org.exoplatform.social.version>6.0.x-SNAPSHOT</org.exoplatform.social.version>
-    <org.exoplatform.calendar.version>6.0.x-SNAPSHOT</org.exoplatform.calendar.version>
+    <org.exoplatform.commons.version>6.2.x-SNAPSHOT</org.exoplatform.commons.version>
+    <org.exoplatform.social.version>6.2.x-SNAPSHOT</org.exoplatform.social.version>
+    <org.exoplatform.jcr.version>6.2.x-SNAPSHOT</org.exoplatform.jcr.version>
```
Comment (hbenali) [inline pom.xml:30]:
```suggestion
    <addon.exo.jcr.version>6.2.x-SNAPSHOT</addon.exo.jcr.version>
```

File: component/pom.xml [hunk 3]
```diff
@@ -40,9 +45,23 @@
...
-
-    </dependencies>
+		<dependency>
+			<groupId>javax.jcr</groupId>
+			<artifactId>jcr</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.exoplatform.jcr</groupId>
+			<artifactId>exo.jcr.component.core</artifactId>
+			<version>${org.exoplatform.jcr.version}</version>
```
Comment (hbenali) [inline component/pom.xml:55]:
```suggestion
			<version>${addon.exo.jcr.version}</version>
```

File: component/pom.xml [hunk 4]
```diff
@@ -40,9 +45,23 @@
...
+			<scope>compile</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.exoplatform.jcr</groupId>
+			<artifactId>exo.jcr.component.ext</artifactId>
+			<version>${org.exoplatform.jcr.version}</version>
```
Comment (hbenali) [inline component/pom.xml:61]:
```suggestion
			<version>${addon.exo.jcr.version}</version>
```

**Review by hbenali — COMMENTED** [review]
Please rename the JCR's property name to be in harmony with the release job ie [commit](https://github.com/exoplatform/swf-release-manager-catalog/commit/74aa611b9d4da1e2685f3724e99433ca4ca74efc#diff-99468a73f6c73b75ada197efea6ecc65e550fcb48eca599e201c1c28a1eaa6b8R488)

File: pom.xml [hunk 5]
```diff
@@ -60,13 +61,6 @@
...
         <scope>import</scope>
       </dependency>
-      <dependency>
-        <groupId>org.exoplatform.calendar</groupId>
-        <artifactId>calendar-service</artifactId>
-        <version>${org.exoplatform.calendar.version}</version>
-        <type>pom</type>
-        <scope>import</scope>
-      </dependency>
       <dependency>
```
Comment (hbenali) [inline pom.xml:64]: Please add here
`      <dependency>
        <groupId>org.exoplatform.jcr</groupId>
        <artifactId>jcr-parent</artifactId>
        <version>${addon.exo.jcr.version}</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
`

File: pom.xml (hunk 2 above)
Comment (Rdidmohamedelhadi) [reply pom.xml]: ok !

File: component/pom.xml (hunk 3 above)
Comment (Rdidmohamedelhadi) [reply component/pom.xml]: we cannot use provided in our case

**Review by hbenali — APPROVED** [review]

**Review by ahamdi — APPROVED** [review]

**Pull request merged**